### ISSUE TITLE
New CST 2.0 autoscale example does not set admin password in DO

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-azure-arm-templates#207]
## Do you already have an issue opened with F5 support?
No

## Description
Admin password not set in DO. When looking at DO example files in autoscale exaxmple for new CST 2.0, there is no password set in DO example. I can add to my own example and it will work.

However, the verbiage in the README for the autoscale example says this:
```
If you provide a value for the newPassword template input parameter, the value is stored in an Azure KeyVault secret. The secret is read securely at deployment time and injected into the sample F5 Declarative Onboarding declaration. When deployment is complete, you can authenticate using the admin account using this password.
```

If you state those actions in the README, then the deployment should result in an admin password being set in the DO declaration. This is not happening.

## Template
https://github.com/F5Networks/f5-azure-arm-templates/tree/master/examples/autoscale

## Severity Level
4

**Comment by shyawnkarim** [comment]
Thanks for reporting this issue. I've filed a bug, internal ID ESECLDTPLT-2227, to get this fixed.

**Comment by shyawnkarim** [comment]
Closing due to age. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-azure-arm-templates-v2).
